proto/abci/types.proto

```diff
@@ -280,6 +280,7 @@ message ConsensusParams {
   BlockParams                      block     = 1;
```

Comment:
Not related to this PR but is there anything bad that could happen if we expose BlockItoams to the application? Currently, we have duplicate consensuses params because of this one field. 

will create an issue to discuss